### ISSUE TITLE
Red line and unsupported cameras pixel 6pro

### ISSUE THREAD

**Issue by OverCloker** [mirsadm/motioncam#104]
When using the app, the red bar on the left side of the screen flashes. No access to 4x lens and wide angle. 
![Screenshot_20211231-151143](https://user-images.githubusercontent.com/96914976/147825874-2be9537b-bf86-423c-9f23-d27cab25ff37.png)
![Screenshot_20211231-151034](https://user-images.githubusercontent.com/96914976/147825881-378c74f7-de60-447c-a8e0-2fb888e5d384.png)
![Screenshot_20211231-153334](https://user-images.githubusercontent.com/96914976/147825919-db31a9cb-e124-4374-bc14-f1bd254d40ba.png)

**Comment by mirsadm** [comment]
Does the red line show up in photos/RAW video?

**Comment by nocantsin** [comment]
Chiming in here: no, the red line doesn't appear in the videos. But it's the same issue I reported earlier, that only the main rear camera of Google Pixel phones is recognized/supported by Motion Cam.

**Comment by LuckyBastard73** [comment]
Today I asked if somebody could share images from the Pixel 6 Pro on a major German android help forum (because all images I saw form this device hav not really ben the hit).
Two guys there told me that the app crashes on their devices (8.0.6 and 8.0.7 latest beta).
Only one module sees to work (main camera?) and one got a crash when returning from the settings.

https://www.android-hilfe.de/forum/google-pixel-6-pro.4003/sample-bilder-von-der-motioncam.995690.html#post-12889947

**Comment by mirsadm** [comment]
yes there are some issues with the Pixel 6. The phone itself seem very buggy at the moment but I'm not discounting the fact that maybe there is something I can do within the app to fix these issues. Unfortunately with these device specific issues it's hard to fix it unless I can test on the phone itself.
